webapp-django/crashstats/crashstats/utils.py

```diff
@@ -255,6 +255,25 @@ def _json_clean(value):
     return value.replace("</", "<\\/")
 
 
+def strip_leading_zeros(text):
+    """Strips leading zeros from a hex string.
+
+    Example:
+
+    >>> strip_leading_zeros("0x0000000000032ec0")
+    "0x32ec0"
+
+    :param text: the text to strip leading zeros from
+
+    :returns: hex string with leading zeros stripped
+
+    """
+    try:
+        return hex(int(text, base=16))
+    except (ValueError, TypeError):
+        return text
```

Comment:
This is copied from signature generation code. Though, we can probably import it. I'll fix that now.